R/app.R

```diff
@@ -90,13 +90,20 @@ shinyApp <- function(ui=NULL, server=NULL, onStart=NULL, options=list(),
     enableBookmarking(bookmarkStore)
   }
 
+  # Always handle /session URLs dynamically. This is useful just in case there
+  # is user code that adds "/" as a static path.
+  staticPaths <- list(
+    "/session" = excludeStaticPath()
+  )
+
   # Store the appDir and bookmarking-related options, so that we can read them
   # from within the app.
   shinyOptions(appDir = getwd())
   appOptions <- consumeAppOptions()
 
   structure(
     list(
+      staticPaths = staticPaths,
```

Comment:
Why are we adding this hardcoded `staticPaths` to the shiny app object? I don't see where we are reading this?